test/compatibility-test/testdata/standard.oam.dev_autoscalers.yaml

```diff
@@ -0,0 +1,143 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.4
+  creationTimestamp: null
+  name: autoscalers.standard.oam.dev
```

Comment:
does the CRD really need in test? And don't forget to remove them after their controllers are moved out